classes/book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, author
+  attr_reader :rentals
+
```

Comment:
- [ ] Good updating your Book class 👏 , as for the attribut accessory you have used `name` as a simple variable. To avoid undefined expeption , Kindly use `:name` with the colon (:)